wamr-compiler/main.c

```diff
@@ -338,6 +344,11 @@ main(int argc, char *argv[])
         else if (!strcmp(argv[0], "--disable-llvm-intrinsics")) {
             option.disable_llvm_intrinsics = true;
         }
+        else if (!strncmp(argv[0], "--enable-builtin-intrinsics=", 28)) {
+            if (argv[0][28] == '\0')
+                PRINT_HELP_AND_EXIT();
+            option.builtin_intrinsics = argv[0] + 28;
```

Comment:
The help prompts `It only takes effect when --disable-llvm-intrinsics is set`, is the `aot_intrinsic_fill_capability_flags` function called only when `disable-llvm-intrinsics`?

Reply:
Yes